Classes/ViewHelpers/Uri/AjaxActionViewHelper.php

```diff
@@ -29,6 +29,9 @@
  * index.php?id=123&tx_typoscriptrendering[context]={"record":"tt_content_123","path":"tt_content.list.20.myextension_plugin"}&tx_myextension_plugin[action]=show&tx_myextension_plugin[controller]=Standard&cHash=xyz
  * (depending on the current page and your TS configuration)
  * </output>
+ *
+ * Please note that this only works for plugins that are using the PLUGIN_TYPE_PLUGIN type,
+ * not for PLUGIN_TYPE_CONTENT_ELEMENT plugins.
```

Comment:
> These are constants from the \TYPO3\CMS\Extbase\Utility\ExtensionUtility class

Hmkay. This is misleading nevertheless IMHO. 

> With PLUGIN_TYPE_CONTENT_ELEMENT, the output will be completely empty (and the corresponding controller action never gets called)

Can you open a ticket and provide some more information?
Especially the generated link is crutial to analyze the issues you face.

In general, there is no such limitation you describe, but there are indeed differences.